24/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM debian:bullseye-slim
+
+LABEL maintainer.0="João Fonseca (@joaopaulofonseca)" \
+  maintainer.1="Pedro Branco (@pedrobranco)" \
+  maintainer.2="Rui Marinho (@ruimarinho)"
+
+RUN useradd -r bitcoin \
+  && apt-get update -y \
+  && apt-get install -y curl gnupg gosu \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+ARG TARGETPLATFORM
+ENV BITCOIN_VERSION=24.0
```

Comment:
Should be 24.0.1 now

Reply:
that would be a new branch/pr/dockerfile for that

Reply:
No, 24.0.0 was not officially announced because of a bug which was fixed in 24.0.1. There should not be a docker image released for 24.0.0.

Reply:
Ah my bad, I am following you now. For ref: https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-24.0.md.

I will work on getting this updated for v24.0.1. Thanks for pointing out!